src/backend/parser/Make.js

```diff
@@ -0,0 +1,26 @@
+var JisonLex = require('jison-lex');
+var Jison = require('jison');
+var fs = require('fs');
+var path = require('path');
+
+var dest = path.resolve(process.argv[2]); // first command-line argument
+
+try { // create directory if it does not exist
+    fs.mkdirSync(dest);
+} catch(e) {
+    if (e.code !== 'EEXIST' ) throw e;
```

Comment:
I'd rather use fs.existsSync for this check; much more readable, no need to rethrow error, and exceptions are rather designed to be used in exceptional situations. This one is not exceptional.